docs/bigtable-table-api.rst

```diff
@@ -0,0 +1,164 @@
+interact with individual tables, groups of tables or column families within
+a table.
+
+List Tables
+-----------
+
+If you want a comprehensive list of all existing tables in a cluster, make a
+`ListTables`_ API request with
+:meth:`Cluster.list_tables() <gcloud.bigtable.cluster.Cluster.list_tables>`:
+
+.. code:: python
+
+    tables = cluster.list_tables()
+
+Table Factory
+-------------
+
+To create a :class:`Table <gcloud.bigtable.table.Table>` object:
+
+.. code:: python
+
+    table = cluster.table(table_id)
+
+Even if this :class:`Table <gcloud.bigtable.table.Table>` already
+has been created with the API, you'll want this object to use as a
+parent of a :class:`ColumnFamily <gcloud.bigtable.column_family.ColumnFamily>`
+or :class:`Row <gcloud.bigtable.row.Row>`.
+
+Create a new Table
+------------------
+
+After creating the table object, make a `CreateTable`_ API request
+with :meth:`create() <gcloud.bigtable.table.Table.create>`:
+
+.. code:: python
+
+    table.create()
+
+If you would to initially split the table into several tablets (Tablets are
+similar to HBase regions):
+
+.. code:: python
+
+    table.create(initial_split_keys=['s1', 's2'])
+
+Delete an existing Table
+------------------------
+
+Make a `DeleteTable`_ API request with
+:meth:`delete() <gcloud.bigtable.table.Table.delete>`:
+
+.. code:: python
+
+    table.delete()
+
+Rename an existing Table
+------------------------
+
+Though the `RenameTable`_ API request is listed in the service
+definition, requests to that method return::
+
+    BigtableTableService.RenameTable is not yet implemented
+
+We have implemented :meth:`rename() <gcloud.bigtable.table.Table.rename>`
+but it will not work unless the backend supports the method.
+
+List Column Families in a Table
+-------------------------------
+
+Though there is no **official** method for retrieving `column families`_
+associated with a table, the `GetTable`_ API method returns a
+table object with the names of the column families.
+
+To retrieve the list of column families use
+:meth:`list_column_families() <gcloud.bigtable.table.Table.list_column_families>`:
+
+.. code:: python
+
+    column_families = table.list_column_families()
+
+Column Family Factory
+---------------------
+
+To create a
+:class:`ColumnFamily <gcloud.bigtable.column_family.ColumnFamily>` object:
+
+.. code:: python
+
+    column_family = table.column_family(column_family_id)
+
+There is no real reason to use this factory unless you intend to
+create or delete a column family.
+
+In addition, you can specify an optional ``gc_rule`` (a
+:class:`GarbageCollectionRule <gcloud.bigtable.column_family.GarbageCollectionRule>`
+or similar):
+
+.. code:: python
+
+    column_family = table.column_family(column_family_id,
+                                        gc_rule=gc_rule)
+
+This rule helps the backend determine when and how to clean up old cells
+in the column family.
+
+See the :doc:`Column Families doc <bigtable-column-family>`
```

Comment:
"See the Column Families docs" would read a little better as "See Colum Families"